src/coverlet.msbuild.tasks/coverlet.msbuild.tasks.csproj

```diff
@@ -21,7 +21,7 @@
     <RepositoryUrl>https://github.com/tonerdo/coverlet</RepositoryUrl>
     <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
     <TargetsForTfmSpecificContentInPackage>$(TargetsForTfmSpecificContentInPackage);PackBuildOutputs</TargetsForTfmSpecificContentInPackage>
-    <ContentTargetFolders>build\$(TargetFramework)\</ContentTargetFolders>
+    <ContentTargetFolders>build\</ContentTargetFolders>
```

Comment:
💡 This does not need a trailing `\`.

🔗 https://github.com/NuGet/NuGet.Client/blob/23b76fef368db1df85d4ff5418bb66c25c499c29/src/NuGet.Core/NuGet.Build.Tasks.Pack/NuGet.Build.Tasks.Pack.targets#L38

```suggestion
    <ContentTargetFolders>build</ContentTargetFolders>
```